bin/dkr-build-test-container

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
I thought about using make.  Decided against on grounds:

1) the repo already uses rake instead of make.  Could have 2 make commands
2) I read make only supports sh level functionality and I wanted bash features
3) make's design centre is around automated update of things show source files change - which is not what is needed here.

Not a die in the ditch sorta thing.

If we have a policy of using make for things (which we don't at present) happy to change - and this could be the first to be updated.

Reply:
It's not the dependency functionality that's significant. It pulls all you operations together in once place so as not to have a myriad of scripts and need to jump around the directory tree. 
As for the shell functions I  don't see any that are needed. Compare with my pr the makefile paired with dockerfile make it much easier to follow

Reply:
I have added the makefile.  It still uses scripts.